rss.py

```
@@ -0,0 +1,97 @@
+    link_blacklist = ["/news/", "/announcement/", "/blog"]
+    title_blacklist = ["weekly pack"]
+    link_lower = entry["link"].lower()
+    title_lower = entry["title"].lower()
+
+    if any(keyword in link_lower for keyword in link_blacklist):
+        return False
+    if any(keyword in title_lower for keyword in title_blacklist):
+        return False
+    return True
+
+def download_comic(comic_info: dict) -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)